scripts/ci.sh

```diff
@@ -56,11 +56,11 @@ elif [ "$TEST" = "e2e_ganache" ]; then
   npm run test:e2e:publish
   npm run test:e2e:ganache:core
 
-elif [ "$TEST" = "e2e_gnosis_dex" ]; then
-
-  npm run test:e2e:publish
-  npm run test:e2e:gnosis:dex
-
+#elif [ "$TEST" = "e2e_gnosis_dex" ]; then
+#
+#  npm run test:e2e:publish
+#  npm run test:e2e:gnosis:dex
+#
```

Comment:
You don't need to comment these, as it will not be executed by CI server anyway.